be/test/storage/tablet_mgr_test.cpp

```diff
@@ -46,6 +46,7 @@
 #include "storage/olap_common.h"
 #include "storage/rowset/column_iterator.h"
 #include "storage/rowset/column_reader.h"
+#include "storage/rowset/metadata_cache.h"
```

Comment:
Why add the header?

Reply:
We need this, otherwise ` _tablet_mgr = std::make_unique<TabletManager>(1);` will have some trouble.